demo/Dockerfile

```diff
@@ -0,0 +1,21 @@
+FROM openjdk:11
```

Comment:
Is this your dockerfile, or one I was mucking about with?....if it's mine, then probably delete it, as yours is almost certainly much better?

Reply:
removed all docker-compose/Dockerfile not used